streaming/java/streaming-runtime/src/main/java/io/ray/streaming/runtime/master/scheduler/JobSchedulerImpl.java

```diff
@@ -153,13 +153,13 @@ public boolean startWorkers(ExecutionGraph executionGraph) {
    */
   protected Map<ExecutionVertex, JobWorkerContext> buildWorkersContext(
       ExecutionGraph executionGraph) {
-    RayActor masterActor = jobMaster.getJobMasterActor();
+    RayActor<JobMaster> masterActor = jobMaster.getJobMasterActor();
 
     // build workers' context
     Map<ExecutionVertex, JobWorkerContext> needRegistryVertexToContextMap = new HashMap<>();
     executionGraph.getAllExecutionVertices().forEach(vertex -> {
-      JobWorkerContext ctx = buildJobWorkerContext(vertex, masterActor);
-      needRegistryVertexToContextMap.put(vertex, ctx);
+      JobWorkerContext context = buildJobWorkerContext(vertex, masterActor);
+      needRegistryVertexToContextMap.put(vertex, context);
```

Comment:
What does `needRegistryVertexToContextMap` mean? BTW,  `Registry` is a noun

Reply:
Use 'vertexToContextMap' instead.